tests/lib/nested.sh

```diff
@@ -76,10 +76,15 @@ nested_wait_vm_ready() {
         fi
 
         # Check during $limit seconds that the serial log is growing
+        # Consider the case then the log is not growing but the ssh can be stablished
         # shellcheck disable=SC2016
-        if ! retry -n "$log_limit" --wait 1 --env serial_log="$serial_log" --env output_lines="$output_lines" \
+        if retry -n "$log_limit" --wait 1 --env serial_log="$serial_log" --env output_lines="$output_lines" \
```

Comment:
IDK if we should even have this check here. Perhaps it was overly optimistic to expect that the serial output will be changing. From recent observations, I often saw the case where nothing was really changing on the serial, as tty0/tty1 is used as the console.

Reply:
right, I saw the same. I can remove this and just leave the checks for infinite loops